src/containers/BooksForm.js

```diff
@@ -1,8 +1,22 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { createBook } from '../actions/index';
 
-const BooksForm = () => {
-  const handleSubmit = e => {
+const BooksForm = (props) => {
+  const [title, setTitle] = React.useState(null);
+  const [category, setCategory] = React.useState(null);
```

Comment:
- [ ] Instead of null we can set the default value to be one of the categories"
```JS
const [category, setCategory] = React.useState('action');
```